app/server/utils/user.py

```diff
@@ -662,3 +663,32 @@ def get_user_by_phone(phone: str, region: str, should_raise=False) -> Optional[U
             raise Exception('User not found.')
         else:
             return None
+
+
+def transfer_usages_for_user(user: User) -> List[TransferUsage]:
```

Comment:
this is now the method used by the state machine. it outputs *all* transfer usages, sorted by user's most used (based on that map of usage to count), then the defaults, then everything else

Reply:
if we don't output all transfer usages, a user could only ever see the defaults since those are the only ones that show up until a user sends a different kind... but they don't have a way to send a different kind aside from default in that case